module-4/backend/week_four.md

```diff
@@ -5,15 +5,51 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's your favorite project management tool?
+  
+  Pivotal Tracker
+  
 2. Why do we create staging environments?
+
+  To test out our code before we merge it into production
```

Comment:
we typically push our master branch to staging before we push it to production 👍